regression/tests/whitelabel/test_user_account.py

```diff
@@ -0,0 +1,149 @@
+"""
+Tests for new users using Otto
+"""
+import uuid
+from unittest import skip
+
+from regression.tests.helpers.api_clients import GuerrillaMailApi
+from regression.tests.whitelabel.white_label_tests_base import (
+    WhiteLabelTestsBaseClass
+)
+from regression.pages.whitelabel.inactive_account import InactiveAccount
+from regression.pages.whitelabel.reset_password_page import (
+    ResetPassword,
+    ResetPasswordComplete
+)
+
+from regression.tests.helpers.utils import get_white_label_registration_fields
+from regression.pages.whitelabel.const import PASSWORD
+from regression.pages.whitelabel.activate_account import ActivateAccount
+
+
+class TestUserAccount(WhiteLabelTestsBaseClass):
+    """
+    User Accounts Tests
+    """
+
+    def setUp(self):
+        """
+        Initialize all page objects
+        """
+        super(TestUserAccount, self).setUp()
+        self.inactive_account = InactiveAccount(self.browser)
+        self.reset_password_complete = ResetPasswordComplete(self.browser)
+        user_name = str(uuid.uuid4().node)
+        self.temp_mail = GuerrillaMailApi(user_name)
+
+    @skip(
+        'This test requires access to gmail,'
+        'currently we are using an alternate test'
+    )
+    def test_00_activate_account(self):  # TE-2044
```

Comment:
nit: we should delete the code for these skipped test cases and close the TE-2044 ticket if we don't need to revive them because we are using the test_activate_account_and_reset_password flow instead.